packages/loader/container-loader/src/protocol.ts

```diff
@@ -78,7 +82,7 @@ export class ProtocolHandler extends ProtocolOpHandler implements IProtocolHandl
 	): IProcessMessageResult {
 		// Check and report if we're getting messages from a clientId that we previously
 		// flagged as shouldHaveLeft, or from a client that's not in the quorum but should be
-		if (message.clientId != null) {
+		if (message.clientId !== null) {
```

Comment:
This is behavior changing, `undefined` used to go into the block and it won't anymore. I know clientId is a weird case where we actually use `null`, but probably safer to not change this here.